deepface/DeepFace.py

```diff
@@ -45,7 +46,7 @@
 # -----------------------------------
 
 
-def build_model(model_name):
+def build_model(model_name: str) -> Any:
```

Comment:
build_model returns one of these objects:

- keras.engine.functional.Functional
- deepface.basemodels.DlibResNet.DlibResNet
- deepface.basemodels.SFace.SFaceModel

So, this should be typed as

```python
from typing import Union
from keras.engine.functional import Functional
from deepface.basemodels import DlibResNet, SFace

def build_model(model_name: str) -> Union[Functional, DlibResNet.DlibResNet, SFace.SFaceModel]:
    pass
```

We should then get rid of Any import in this module.

Reply:
Or you may consider to use Functional and any. Because except 2 models, this function returns Functional.

```python
Union[Functional, Any]
```